pkg/arvo/sys/lull.hoon

```diff
@@ -1163,6 +1163,9 @@
         ::    not allowed.
         ::
         [%bound accepted=? =binding]
+        ::  notification that the clay file associated with url has changed
+        ::
+        [%cache-changed url=@t =beam]
```

Comment:
Is `beam` the right type to use for this? It seems to have the information we care about, but I wasn't sure.